dashboard/projects/templates/projects/project.html

```diff
@@ -14,27 +14,47 @@
 
 {% block content %}
 
-<h1>{{ project }}</h1>
+  <div id="container">
 
-<div id="container">
-  <div class="main" id="main">
-    <main id="content-start" class="content">
-      <div id="content" class="colM">
-        <div id="content-main">
+    <nav class="navigation">
+      <ul>
+        <li><a href="/">Dashboard</a></li>
+        {% if user.is_authenticated %}
+          <li><a href="{% url 'admin:index' %}">Admin</a></li>
+          <li>Logged in as {{ user.username }}</li>
+          <li>
+            <form action="{% url 'logout' %}" method="post">
```

Comment:
How about sending the user back to the project page when they log out?

```suggestion
            <form action="{% url 'logout' %}?next={{ request.path }}" method="post">
```